ssl/record/methods/dtls_meth.c

```diff
@@ -742,11 +742,15 @@ int dtls_post_encryption_processing(OSSL_RECORD_LAYER *rl,
 
 static size_t dtls_get_max_record_overhead(OSSL_RECORD_LAYER *rl)
 {
+    int blksz = 0;
     size_t blocksize = 0;
 
     if (rl->enc_ctx != NULL &&
-        (EVP_CIPHER_CTX_get_mode(rl->enc_ctx) == EVP_CIPH_CBC_MODE))
-        blocksize = EVP_CIPHER_CTX_get_block_size(rl->enc_ctx);
+        (EVP_CIPHER_CTX_get_mode(rl->enc_ctx) == EVP_CIPH_CBC_MODE)) {
+        blksz = EVP_CIPHER_CTX_get_block_size(rl->enc_ctx);
+        assert(blksz >= 0);
```

Comment:
Shouldn't we instead propagate this error to the caller (there is only one)?

Reply:
Also a block size of 0 is really an error.  Stream ciphers have block size of 1.

Reply:
Updated - the propagation required a few more changes